src/components/cards/NFTOffersCard/Offer.tsx

```diff
@@ -119,7 +130,9 @@ export const Offer = ({
     case SortCriterion.DateCreated:
       if (isExpiring) {
         secondaryTextColor = 'red';
-        secondaryText = getFormattedTimeQuantity(timeRemaining);
+        secondaryText = timeRemaining
```

Comment:
we purge expired offers from the RQ cache every minute, however in that minute they can still remain visible to the user. instead of displaying "0m" remaining, display "Expired"